.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,59 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '15 21 * * 6'
+  push:
+    branches: [ "main" ]
```

Comment:
Could you add something like

```yml
  pull_request:
    paths:
      - '.github/workflows/scorecards.yml'
 ```

so we can see the Action in *action* in this PR (and in PRs which would eventually update the Action configuration)? Assuming it works when run in a PR context.

Reply:
it is not officially supported yet, but I can enable it

Reply:
Done 😄